### PR TITLE
Minor cleanups, clarifications

### PR DESCRIPTION
**Pull request by 130s**
- <s>	Remove repository specific files (.travis.yml, .gitignore)</s>
- Added usage of `*.rosinstall` files.
- <s>I'm not sure if it was intentional to use `moveit.rosinstall` without prefixing dot.</s>

### CODE REVIEW

**Comment by davetcoleman** [comment]
``moveit.rosinstall`` is user-facing - its used by developers to download all the source repos at once - so I think it should not be hidden / prefixed with a dot. This is how it [already is for Moveit](https://github.com/ros-planning/moveit_docs) and I found that format [elsewhere](https://erlerobotics.gitbooks.io/erlerobot/content/en/ros/tutorials/rosinstall.html) just now

File: .moveit.rosinstall [hunk 1]
```diff
@@ -1,3 +1,5 @@
+# This file is primarilly intended for the users who want to build MoveIt! from source.
+# Used with wstool, users can download source of all packages of MoveIt!.
```
Comment (davetcoleman) [inline .moveit.rosinstall]: i love the comments!

**Comment by davetcoleman** [comment]
Changes to anything inside the packages (like deleting gitignore files) should wait until after "merge day" (when we switch the master to this repo) or should be reflected in the automated script  that we will have to run again on merge day. I added that commit [here](https://github.com/davetcoleman/moveit_merge/commit/f0a5601a8b3cdcedd75c332b1b5009272adc2667)

**Comment by 130s** [comment]
Makes sense. Updated.

**Comment by davetcoleman** [comment]
Better script update (untested): https://github.com/davetcoleman/moveit_merge/commit/659c1934382277bf260c692cef7a7129a66e7ee2

**Pull request merged**